koap/js_tests/helpers.js

```diff
@@ -0,0 +1,7 @@
+global.byteToHexString = (byte) => {
```

Comment:
Global modifications may bleed into the rest of the application. These can be defined as functions that can be imported into your tests.

Reply:
Well, I might not need these helpers at all.